### PR TITLE
Fix XML syntax errors in schemaImporterExtensions docs

### PR DESCRIPTION
**Pull request by adamgauthier**
## Summary

- Fix `SchemaImporterExtension` -> `schemaImporterExtensions`
- Fix missing '<' in `system.xml.serialization` closing tag
- Fix serializaiton -> serialization
- Fix `sxml` -> `xml`

### CODE REVIEW

**Comment by rpetrusha** [comment]
Thanks, @louistio, for spotting this error. There is, though, one more inconsistency: the **Child Elements** section lists the name of this element as \<xmlSchemaImporterExtensions>, as does the article for the [\<add> Element>](https://docs.microsoft.com/en-us/dotnet/standard/serialization/add-element-for-xmlschemaimporterextensions). @ViktorHofer, can you confirm that the name of the element is \<schemaImporterExtensions> rather than \<xmlSchemaImporterExtensions>?

**Comment by louistio** [comment]
@rpetrusha Good observation! I'll wait on the answer and make the changes accordingly in this PR.

**Review by BillWagner — DISMISSED** [review]
Thanks for spotting this and fixing it @louistio We appreciate it.

I've reviewed the changes, and I'll :shipit: now.  

Thanks again for contributing to docs!

**Comment by ViktorHofer** [comment]
Sorry not my area but fortunately I know the owners ;) @krwq, @pjanotti

**Comment by pjanotti** [comment]
This is actually related to xml serialization and related tools. @huanwu, @Lxiamail, and @zhenlan are in a better position to answer @rpetrusha question.

**Comment by rpetrusha** [comment]
I apologize for the delay in getting back to you, @louistio. It turns out that both `<schemaImporterExtensions>` and `<xmlSchemaImporterExtensions>` are supported. However, we should standardize on one variant consistently in the documentation. Could you change line 25 so that it reads "\<schemaImporterExtensions>" instead of "\<xmlSchemaImporterExtensions>"? Also, could you modify the [\<add> Element for \<xmlSchemaImporterExtensions>](https://github.com/dotnet/docs/blob/master/docs/standard/serialization/add-element-for-xmlschemaimporterextensions.md) so that "\<schemaImporterExtensions>" replaces "\<xmlSchemaImporterExtensions>" and "\<XmlSchemaImporterExtensions>" throughout the document?  It's not necessary to rename the file. If you prefer, you could also just make the additional change to the single file in this PR, and we could modify the **\<add> Element for \<xmlSchemaImporterExtensions>** article.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
See my previous comment for requested changes, @louistio.

**Comment by louistio** [comment]
Hey @rpetrusha, thank you for the clarification, I also agree the docs should standardize on one variant, which is not the case currently. Would you mind if I make these additional changes in another PR as I feel they are distinct from this PR that aims to specifically correct xml syntax errors?

**Comment by rpetrusha** [comment]
No, that's fine, @louistio.

**Comment by louistio** [comment]
Hey @rpetrusha, I disagree as I see both as separate changes and would prefer they are done as two different PRs, it makes more sense for the git history, rather than bundling them together. I created a PR for the consistency issue over at #6807, please tell me what you think.

**Comment by rpetrusha** [comment]
Thanks, @louistio. If you'd like to do them as separate PRs, that's fine. It probably makes the most sense to merge them together, though, so I'll approve this one but won't merge until #6807 is approved.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
On second thought, there's no good reason not to merge this now, @louistio. It should be live on docs.microsoft.com tomorrow.

**Pull request merged**
